boot/assets_test.go

```diff
@@ -66,6 +66,10 @@ func checkContentGlob(c *C, glob string, expected []string) {
 
 func (s *assetsSuite) uc20UpdateObserver(c *C) (*boot.TrustedAssetsUpdateObserver, *asserts.Model) {
 	uc20Model := makeMockUC20Model()
+	// checked by TrustedAssetsUpdateObserverForModel
+	s.stampSealedKeys(c, dirs.GlobalRootDir)
+	// checked by resealKeyToModeenv, under this rootdir
```

Comment:
Just to be clear these are the same physical file on a real system, just that this helper function gets called from multiple different situations (i.e install mode and run mode) and thus it's easier to just always create both files than otherwise track which file each situation actually will look at.  In other words, we create both files here for sale of simplicity but each situation will only ever look at one of these files, is that correct?

Reply:
actually, as I thought for a moment the InstallHost* dirs don't exist in run-mode (though mvo5 checked they did but probably was confused by data vs ubuntu-data), so I have to untangle this

Reply:
Indeed, sorry for this. I do remember our change from ubuntu-data -> data in run mode :/